lib/resque/failure/blacklist_exceptions_registrar.rb

```diff
@@ -0,0 +1,32 @@
+module Resque
+  module Failure
+    # A module to access blacklisted exceptions
+    # on a class level
+    module BlacklistExceptionsRegistrar
+      def self.included(base)
+        base.extend ClassMethods
+      end
+
+      # Class methods for accessing
+      # blacklisted exceptions
+      module ClassMethods
+        # Set a list of blacklisted exceptions
+        #
+        # The method sets a list of exceptions which will
+        # not be pushed to the failed queue when a job fails and it's
+        # exception is included to the blacklisted ones.
+        #
+        # @param [Array<String>] exceptions the list of exceptions to blacklist
+        def set_blacklisted_exceptions(exceptions)
+          @blacklist = exceptions
+        end
+
+        def get_blacklisted_exceptions
+          return @blacklist if defined?(@blacklist) && @blacklist.size
```

Comment:
We can drop the `&& @blacklist.size` part from the conditional. If the variable is defined, return its contents.